clar2wasm/src/words/constants.rs

```diff
@@ -60,3 +60,63 @@ impl ComplexWord for DefineConstant {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use clarity::vm::types::{ListData, ListTypeData, SequenceData};
+    use clarity::vm::Value;
+
+    use crate::tools::TestEnvironment;
+
+    #[test]
+    fn define_constant_const() {
+        let mut env = TestEnvironment::default();
```

Comment:
If you're not using the extra functionality of TestEnvironment, just use `evaluate` directly

Reply:
Also, why not use crosscheck here as well?

Reply:
I will be fair, I completely cut and paste that from @csgui, I don't even know what it looks like :D
I'll change the tests to crosscheck

Reply:
@krl @Acaccia It was an old implementation before `crosscheck` capability. :-)

Reply:
Fixed in 2d75343